cdc/log.cc

```diff
@@ -306,10 +323,13 @@ static schema_ptr create_log_schema(const schema& s, std::optional<utils::UUID>
                         return type.freeze();
                     }
                 ));
-
-                type = tuple_type_impl::get_instance({ /* op */ data_type_for<column_op_native_type>(), /* value */ type->freeze()});
+                type = type->freeze();
             }
             b.with_column(log_data_column_name_bytes(column.name()), type);
+            b.with_column(log_data_column_deleted_name_bytes(column.name()), boolean_type);
+            if (type->is_collection()) {
```

Comment:
Is this distinguishing between frozen and non-frozen mutations and adding column only to non-frozen collections?

Reply:
No, it is not. Will fix.